editor.planx.uk/src/@planx/components/MultipleFileUpload/model.test.ts

```diff
@@ -360,3 +372,129 @@ describe("getRecoveredSlots function", () => {
     expect(result[0]).toMatchObject(mockCachedSlot);
   });
 });
+
+describe("getTagsForSlot function", () => {
```

Comment:
Could we add a test case here for a slot without any tags - I'm relying on this function to handle this in #1811 so it would be good to cover this scenario.

Reply:
Good shout, now covered ! Will return `[]` in cases where a slot does not have any tags fyi !